addons/common/initSettings.inc.sqf

```diff
@@ -100,3 +100,21 @@ private _categoryColors = [_category, format ["| %1 |", LLSTRING(subcategory_col
     [[0, 1, 2], [LSTRING(None), LSTRING(progressBarInfoPercentage), LSTRING(progressBarInfoTime)], 2],
     0
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(QTEHoldKey),
+    "LIST",
+    localize LSTRING(QTEHoldKey),
+    "ACE QTE",
+    [["Ctrl", "Alt", "Shift"], ["Ctrl", "Alt", "Shift"], 0],
```

Comment:
These should be stringtable entries. Either ACE or Vanilla have these already defined .

Reply:
I can't seem to find it in ACE how can I check Vanilla?

Reply:
`str_dik_alt, str_dik_shift, str_dik_control`

Reply:
https://brettmayson.github.io/stringtables/